type-scheme.svg

```diff
@@ -0,0 +1,931 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should move this to some `docs/` folder.

Reply:
Nice figure btw :-)